packages/paste-website/src/pages/core/changing-theme.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: Changing the Theme
+description: When using Paste, migrating to a different theme is a simple two step process.
```

Comment:
We describe this as having two steps in the description, but don't delineate what those two steps are in the content. I think we should either clearly say in the content 1. make sure you've got theme.provider 2. change the theme to whatever you want OR just take this part out of the description and say something like "...migrating to a different theme is simple and quick." or "Using Paste's Theme Provider to migrate to a different theme is simple and quick."

Reply:
Oooooh great callout! I updated the description to say "...migrating to a different theme is simple and quick."